storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -235,6 +237,11 @@
   (let [key [component stream]]
     (update-executor-stat! stats :failed key (stats-rate stats))))
 
+(defn bolt-deserialize-time!
+  [^BoltExecutorStats stats component stream deserialize-time]
```

Comment:
I am nervious about this being specific to a bolt, and the spout calling into this.  Clojure is going to have to use reflection to make this work, which is slow.